.github/workflows/config/environment-region-map.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This json could be managed by IaC in to S3 at some point if we get to a point where regions are being added/removed more frequently.